repose-aggregator/components/filters/versioning-filter/src/main/scala/org/openrepose/filters/versioning/VersioningFilter.scala

```diff
@@ -115,6 +108,16 @@ class VersioningFilter @Inject()(@Value(ReposeSpringProperties.NODE.CLUSTER_ID)
       val configurationData = new ConfigurationData(destinations, serviceVersionMappings)
 
       try {
+        def markMeter(name: String): Unit = {
```

Comment:
This seems like a really odd place to define this function. We don't really need to redefine it for each request since it's parameterized anyway, so it can probably be extracted.

Oh, here's an opportunity to show the Scala trick I wanted to use!

So first, pull out the commonalities.

```
val metricRegistry = metricService.getRegistry
val meterName: String => Meter  = metricService.name(
    "org.openrepose.core.filters.Versioning",
    "versioning", // todo: replace "versioning" with filter-id or name-number in sys-model,
    "VersionedRequest",
    _: String
)
```

Then when you want to mark a meter:

```
metricRegistry.meter(meterName(targetOriginService.getMapping.getId)).mark()
```